modules/osdk-ansible-watches-file.adoc

```diff
@@ -0,0 +1,140 @@
+// Module included in the following assemblies:
+//
+// * operators/osdk-ansible.adoc
+
+[id='osdk-ansible-watches-file-{context}']
+= Watches file
+
+The Watches file contains a list of mappings from Custom Resources (CRs),
+identified by its `Group`, `Version`, and `Kind`, to an Ansible role or
+playbook. The Operator expects this mapping file in a predefined location,
+`/opt/ansible/watches.yaml`.
+
+.Watches file mappings
+[cols="3,7",options="header"]
+|===
+|Field
+|Description
+
+|`group`
+|Group of CR to watch.
+
+|`version`
+|Version of CR to watch.
+
+|`kind`
+|Kind of CR to watch
+
+|`role` (default)
+|Path to the Ansible role added to the container. For example, if your `roles`
+directory is at `/opt/ansible/roles/` and your role is named `busybox`, this
+value would be `/opt/ansible/roles/busybox`. This field is mutually exclusive
+with the `playbook` field.
+
+|`playbook`
+|Path to the Ansible playbook added to the container. This playbook is expected
+to be simply a way to call roles. This field is mutually exclusive with the
+`role` field.
+
+|`reconcilePeriod` (optional)
+|The reconciliation interval, how often the role or playbook is run, for a given
+CR.
+
+|`manageStatus` (optional)
+|When `true` (default), the Operator manages the status of the CR generically.
+Set to `false`, the status of the CR is managed elsewhere, by the specified role
+or playbook or in a separate controller.
+|===
+
+.Example Watches file
+[source,yaml]
+----
+- version: v1alpha1 <1>
+  group: foo.example.com
+  kind: Foo
+  role: /opt/ansible/roles/Foo
+
+- version: v1alpha1 <2>
+  group: bar.example.com
+  kind: Bar
+  playbook: /opt/ansible/playbook.yml
+
+- version: v1alpha1 <3>
+  group: baz.example.com
+  kind: Baz
+  playbook: /opt/ansible/baz.yml
+  reconcilePeriod: 0
+  manageStatus: false
+----
+<1> Simple example mapping `Foo` to the `Foo` role.
+<2> Simple example mapping `Bar` to a playbook
+<3> More complex example for the `Baz` kind. Disables re-queuing and manages the
```

Comment:
` Disables re-queuing and manages the CR status` -> `Disables re-queuing and managing the CR status`